app/controllers/users_controller.rb

```diff
@@ -0,0 +1,54 @@
+class UsersController < ApplicationController
+
+  before_action :admin_required!
+
+  def new
+    @user=User.new
```

Comment:
nitpick: spaces

Reply:
Can you give me a hint: Is it something about identation?

Reply:
Ah, sorry: `@user = User.new` (which is more consistent with the rest of the code)

Reply:
done